### PR TITLE
[#891] Revisit Locations

### LINKED ISSUE #891: Revisit Locations Metadata Subfield
We will use DH subject_geographic for this field. The source of subject_geographic in DH is FAST-Geographic, so we can add that as the `identifier`.

### PR DESCRIPTION
**Pull request by sharpattack**
Use subject_geographic to populate locations metadata. closes #891

### CODE REVIEW

**Comment by sharpattack** [comment]
Unfortunately, `subject_geographic` is stored with the text value only. So we are not able to get it's identifier to add.

File: app/models/invenio_rdm_record_converter.rb [hunk 1]
```diff
@@ -157,7 +157,7 @@ def invenio_metadata(gf)
       "formats": gf.mime_type,
       "version": version(gf.content),
       "rights": rights(gf.rights),
-      "locations": {"features": gf.based_near.present? ? gf.based_near.shift.split("', ").map{ |location| {place: location.gsub("'", "")} } : []},
+      "locations": {"features": gf.subject_geographic.present? ? gf.subject_geographic.map{ |location| {place: location.gsub("'", "")} } : []},
```
Comment (fenekku) [inline app/models/invenio_rdm_record_converter.rb:160]: Curious: Why do we have to do `location.gsub("'", "")`? (Other characters to remove?)

File: spec/models/invenio_rdm_record_converter_spec.rb [hunk 2]
```diff
@@ -178,6 +178,7 @@
     subject { invenio_rdm_record_converter.to_json }
 
     it do
+      #binding.pry
```
Comment (fenekku) [inline spec/models/invenio_rdm_record_converter_spec.rb:181]: can remove

**Review by fenekku — APPROVED** [review]
Questions/minor things, once resolved we can merge.

File: app/models/invenio_rdm_record_converter.rb (hunk 1 above)
Comment (sharpattack) [reply app/models/invenio_rdm_record_converter.rb:160]: Ah, good catch. We do not need it, I missed removing this.

File: spec/models/invenio_rdm_record_converter_spec.rb (hunk 2 above)
Comment (sharpattack) [reply spec/models/invenio_rdm_record_converter_spec.rb:181]: Whoops, thank you. 🤦

**Review by Meowcenary — APPROVED** [review]

**Pull request merged**